be/src/exec/vectorized/cross_join_node.cpp

```diff
@@ -577,6 +605,7 @@ pipeline::OpFactories CrossJoinNode::decompose_to_pipeline(pipeline::PipelineBui
     using namespace pipeline;
```

Comment:
In decompose_to_pipeline, no nljoin operators appear?

Reply:
That could change too much code. I could change it later.